R/Sharis/code_chunks.Rmd

```diff
@@ -0,0 +1,170 @@
+---
+title: "Code Chunks"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+## Updating Coverage
+
+If you want to add to the current coverage without having to restate all the other coverage elements using `set_coverage`, these chunks allow you to just add taxonomic coverage and single date temporal coverage. 
+
+### Adding taxonomic coverage
+
+```{r, eval = FALSE}
+# add each new element as a tax object
+tax1 <- new("taxonomicClassification",
+            taxonRankName = new("taxonRankName", "Species"),
+            taxonRankValue = new("taxonRankValue", "Calamagrostis deschampsioides"))
+
+tax2 <- new("taxonomicClassification", 
+            taxonRankName = new("taxonRankName", "Species"),
+            taxonRankValue = new("taxonRankValue", "Carex aquatilis"))
+
+# combine all tax elements into taxonomic coverage object
+taxcov <- new("taxonomicCoverage",
+               taxonomicClassification = c(tax1, tax2))
+
+eml@dataset@coverage@taxonomicCoverage <- c(taxcov)
+
+```
+
+### Adding single date temporal coverage
+
+```{r, eval = FALSE}
+date <- new("singleDateTime",
+           calendarDate = "2011-09-15")
+
+tempcov1 <- new("temporalCoverage",
+               singleDateTime = date)
+
+eml@dataset@coverage@temporalCoverage <- c(tempcov1)
+```
+
+## Data Objects
+### Adding data tables for a whole folder of files with the same attributes
```

Comment:
This section is so awesome!  I just had to do this but forgot you wrote it up until I mostly finished.  In any case, interesting to see slightly different ways of doing the same thing!